src/ppl/src/HungarianMatching.cpp

```diff
@@ -397,7 +397,7 @@ std::vector<uint8_t> HungarianMatching::getTieBreakRank(
 {
   std::vector<uint8_t> rank(costs.size());
   uint8_t ranking = 1;
-  for (int i : sortIndexes(costs, costs)) {
+  for (int i : sortIndexes(costs, costs, costs)) {
```

Comment:
It seems odd to have tie breaking code that won't see any difference in the tie breaking values.

Reply:
In this specific case, I want only the regular sorting, as we don't have any tie break criteria here. Perhaps I should add an overload to this function for clarity?

Reply:
I think that would help